mne/viz/_3d.py

```diff
@@ -1122,8 +1128,23 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
             opacity=0.75, glyph_height=0.25,
             glyph_center=(0., 0., 0.), glyph_resolution=20,
             backface_culling=True)
-    renderer.set_camera(azimuth=90, elevation=90,
-                        focalpoint=(0., 0., 0.), distance=0.6)
+    if view is None:
+        renderer.set_camera(azimuth=90, elevation=90,
+                            focalpoint=(0., 0., 0.), distance=0.6)
+    elif isinstance(view, dict):
+        renderer.set_camera(azimuth=view.get('azimuth', 90),
+                            elevation=view.get('elevation', 90),
+                            focalpoint=view.get('focalpoint',
+                                                (0., 0., 0.)),
+                            distance=view.get('distance', 0.6))
+    else:
+        raise TypeError('view must be an instance of `dict` or `None`: '
+                        '{} was given'.format(type(view)))
+    if isinstance(title, str):
+        renderer.title(title)
+    elif title is not None:
+        raise TypeError('title must be an instance of `str` or `None`: '
+                        '{} was given'.format(type(title)))
```

Comment:
this should be put in a separate private function. If you decide to make the function public

mne.viz.set_camera(renderer or mlab.figure or scene, view)

you don't need to add any parameter for view to the public API.

thoughts someone?

Reply:
maybe a `kwargs` as extra parameters?

Reply:
please no kwargs in public functions if we can